nimble_build_system/cad/shelf.py

```diff
@@ -364,6 +413,9 @@ def generate_assembly_model(self, explode=False):
         # the method dispatch. If I make Python happy, pylint fails.
         # pylint: disable=E1120
         # pylint: disable=W0212
+        # pylint: disable=R0912
+        # pylint: disable=R0915
+        # pylint: disable=E1121
```

Comment:
Probably in future when disabling it would be best to disable by name 

i.e. `disable=too-many-statements` rather than `disable=R0915` as it is more human readable.